py-data.csv

```diff
@@ -32,9 +32,9 @@ https://github.com/alexrudy/supertunnel,4c64e2fedda3382b809a2c3d01c711ec4961f933
 https://github.com/alexrudy/supertunnel,4c64e2fedda3382b809a2c3d01c711ec4961f933,supertunnel/ssh/continuous_test.py::test_ssh_log_line[Host example.com not responding-Action.DISCONNECTED],OD-Brit,Unmaintained,,latest commit 2019/11/21
 https://github.com/aluttik/kitsune,43824cccf46f433a71b30a7febc0e3500b831067,tests/test_kitsune.py::test_kitsune_printer_plain,NIO,Unmaintained,,latest commit 2018/11/19
 https://github.com/aluttik/kitsune,43824cccf46f433a71b30a7febc0e3500b831067,tests/test_kitsune.py::test_kitsune_printer_plain tests/test_kitsune.py::test_kitsune_printer_rainbow,,Unmaintained,,pytest --flake-finder --flake-runs=2;latest commit 2018/11/19
-https://github.com/AshtonUPS/Py-MI-PS,e43c2b4df2b59362c147024840bf541bb767ffd7,src/PyMIPS/tests/register_test.py,NIO,Open,https://github.com/AshtonUPS/Py-MI-PS/pull/23,
-https://github.com/AshtonUPS/Py-MI-PS,2d22327c75bac1b58a4804a61e7a703ecc5ba978,src/PyMIPS/tests/register_test.py::test_setting_to_num,NIO,Opened,https://github.com/AshtonUPS/Py-MI-PS/pull/22,
-https://github.com/AshtonUPS/Py-MI-PS,2d22327c75bac1b58a4804a61e7a703ecc5ba978,src/PyMIPS/tests/register_test.py::test_setting_to_num,OD-Vic,,,
+https://github.com/AshtonUPS/Py-MI-PS,e43c2b4df2b59362c147024840bf541bb767ffd7,src/PyMIPS/tests/register_test.py,NIO,Open,https://github.com/AshtonUPS/Py-MI-PS/pull/23,RepoDeleted
```

Comment:
@darko-marinov Do we want to move RepoDeleted from Notes to the Status column for this and the next test

Reply:
No, let's keep the status as `Open[ed]` if we opened a PR before the repo was deleted.

@Ajitesh27 @Qi-Cui How did the format checker pass with `Open` rather than `Opened`?

Reply:
Do we have a format checker for `py-data.csv`? I only see pr and gr from the CI checker log message
```
Run python format_checker/main.py 93a9[1](https://github.com/TestingResearchIllinois/idoft/pull/1502/checks#step:6:1)de7938fc75903e810cec8c30d9a8f3116f5 f508495bcef88085b3068150b64511fce83e105a
INFO: On file pr-data.csv: There are no changes to be checked
INFO: On file gr-data.csv: There are no changes to be checked
INFO: On file tic-fic-data.csv: There are no changes to be checked
INFO: On file tso-iso-rates.csv: There are no changes to be checked
INFO: On file format_checker/forked-projects.json: There are no changes to be checked
Success: Exiting with code 0 due to no logged errors
```
And indeed, looks like the [checker](https://github.com/TestingResearchIllinois/idoft/blob/de5ed5eaa99466146337257f4e3499e89c3a9c70/format_checker/main.py#L22) only checks for pr and gr. If this is the case, do you want me to make the checker check `py-data`?